middleman-core/middleman-core.gemspec

```diff
@@ -25,39 +25,39 @@ Gem::Specification.new do |s|
   s.add_dependency('parallel')
   s.add_dependency('servolux')
   s.add_dependency('dotenv')
-  s.add_dependency('rgl', ['~> 0.5.3'])
+  s.add_dependency('rgl', ['~> 0.5'])
   s.add_dependency('toml')
   s.add_dependency('webrick')
 
   # Helpers
   s.add_dependency('activesupport', ['>= 5.0.0'])
-  s.add_dependency('padrino-helpers', ['~> 0.14.4'])
-  s.add_dependency('addressable', ['~> 2.3'])
-  s.add_dependency('memoist', ['~> 0.14'])
+  s.add_dependency('padrino-helpers', ['~> 0'])
```

Comment:
Might need to keep this more explicit. Padrino has a habit of breaking changes in their minor version... they've been at 0.x for 12 years :)

Reply:
Fixed by https://github.com/middleman/middleman/pull/2517/commits/be5871e2075f50a007b2fc6ef0846139952716cf